gems/pending/postgres_ha_admin/failover_databases.rb

```diff
@@ -51,6 +51,14 @@ def query_repmgr(connection)
       result
     end
 
+    def symbolize_record_keys(record)
+      new_record = {}
+      record.each do |k, v|
+        new_record[k.to_sym] = v
+      end
+      new_record
```

Comment:
I actually gave that a shot and was getting test failures so I tried it out and got this:

```
irb(main):010:0> h = {"one" => 1, "two" => 2}
=> {"one"=>1, "two"=>2}
irb(main):011:0> h.each_with_object({}) do |k, v, hash|
irb(main):012:1* puts "key = #{k.inspect}"
irb(main):013:1> puts "value = #{v.inspect}"
irb(main):014:1> end
key = ["one", 1]
value = {}
key = ["two", 2]
value = {}
=> {}
```

I wasn't expecting it to work this way either :smile: